vsintegration/src/FSharp.Editor/CodeFix/AddInstanceMemberParameter.fs

```diff
@@ -9,17 +9,26 @@ open System.Collections.Immutable
 open Microsoft.CodeAnalysis.Text
 open Microsoft.CodeAnalysis.CodeFixes
 
+open CancellableTasks
+
 [<ExportCodeFixProvider(FSharpConstants.FSharpLanguageName, Name = CodeFix.AddInstanceMemberParameter); Shared>]
 type internal FSharpAddInstanceMemberParameterCodeFixProvider() =
     inherit CodeFixProvider()
 
     static let title = SR.AddMissingInstanceMemberParameter()
 
+    interface IFSharpCodeFix with
+        member _.GetChangesAsync _ span =
+            cancellableTask {
+                let changes = [ TextChange(TextSpan(span.Start, 0), "x.") ]
+                return title, changes
+            }
```

Comment:
Doesn't need to be CE, you can use CancellableTask.singleton.

Reply:
Cool, thanks!